src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  position: absolute;
+  top: 10px;
+  left: 50px;
```

Comment:
Not critical, but absolute positioning is excessive in this case.
The same can be done with flex settings and paddings.